samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -254,9 +265,18 @@ void main(void)
 {
 	bool run_led_on = true;
 	int err;
+	struct bt_conn_auth_info_cb auth_info_cb = {
```

Comment:
`static struct` - the callbacks must remain valid until unregistered, they shouldn't be placed on stack.
See implementation of `bt_conn_auth_info_cb_register` for details